packages/react-storybook/src/decorators/withFluentProvider.tsx

```diff
@@ -15,6 +15,6 @@ export const withFluentProvider = makeDecorator({
   parameterName: 'theme',
   skipIfNoParametersOrOptions: false,
   wrapper: (storyFn, context) => {
-    return <ProviderWrapper>{storyFn(context)}</ProviderWrapper>;
+    return <ProviderWrapper> {storyFn(context)} </ProviderWrapper>;
```

Comment:
Merge issue?

Reply:
intentional for now. TS was complaining that ProviderWrapper was returning null. The workaround was making sure whitespace was always returned